packages/sdk/package.json

```diff
@@ -49,5 +53,9 @@
   },
   "peerDependencies": {
     "ethers": "^6.3.0"
+  },
+  "dependencies": {
+    "@artblocks/contracts": "^1.0.2",
```

Comment:
It seems like we want our SDK to depend on the latest _published_ NPM package, and it should be a hard dependency instead of a peer dependency. Does that sound right?